test/hummingbot/connector/exchange/bybit/test_bybit_api_order_book_data_source.py

```diff
@@ -200,11 +196,9 @@ def test_get_new_order_book(self, mock_api):
         self.assertEqual(1, len(bid_entries))
         self.assertEqual(50005.12, bid_entries[0].price)
         self.assertEqual(403.0416, bid_entries[0].amount)
-        self.assertEqual(int(resp["result"]["time"]), bid_entries[0].update_id)
         self.assertEqual(1, len(ask_entries))
```

Comment:
why removing the ts check?

Reply:
Because V5 API returns an `Update ID` parameter (`["result"]["u"]`), that was missing from the previous API versions.

```
    "result": {
        "s": "BTCUSDT",
        "a": [
            [
                "65557.7",
                "16.606555"
            ]
        ],
        "b": [
            [
                "65485.47",
                "47.081829"
            ]
        ],
        "ts": 1716863719031,
        "u": 230704,
        "seq": 1432604333,
        "cts": 1716863718905
    },
```

Internally `["result"]["u"]` is used for setting the `update_id` of bid/ask entries

Reply:
I just did add the below assertion for verifying that.

```py
self.assertEqual(int(resp["result"]["u"]), bid_entries[0].update_id)
self.assertEqual(int(resp["result"]["u"]), ask_entries[0].update_id)
```